README.md

```diff
@@ -2,34 +2,196 @@
+
+#### Installing protector in your Laravel project
+
+Install the package via composer.
 
 ```bash
 composer require cybex/laravel-protector
 ```
 
-## Usage
+You can optionally publish the protector config to set the following options
+- fileName: the file name of the database dump
+- baseDirectory: where files are being stored
+- diskName: a dedicated Laravel disk defined in config/filesystems.php. These can point to a specific folder, disk or a  cloud file bucket like AWS S3.
 
-``` php
-To generate a sodium encryption key pair use
-    protector:keys
+```bash
+artisan vendor:publish --tag=protector.config
+```
+
+#### Usage
+
+You can now use the artisan command to write a backup to the protector storage folder.
 
-To generate a Laravel Sanctum authentication token for a user use
-    protector:token <userID>
-    
-You can import a database dump by using
-    protector:import
-    
-You can export a database dump by using
-    protector:export
+```bash
+php artisan protector:export
 ```
 
+By default the file will be stored in storage/protector and have a timestamp in the name. You can also specify the filename.
+
+You could also automate this by
+- installing a cronjob on linux
+- running it when you deploy to your server
+- creating a Laravel Job and queueing it
+
+```bash
+php artisan protector:export --file=storage/database.sql
+```
+
+### Setup for importing the database of a remote server
+
+This package can run on both servers and client machines of the same software repository. You set up authorized developers on the server, and give them the key for their local machine.
+
+#### Installing protector in your Laravel project
+
+Install the package via composer.
+
+```bash
+composer require cybex/laravel-protector
+```
+
+In your User model class, add the following trait.
+```php
+use Laravel\Sanctum\HasApiTokens;
+
+class User extends Authenticatable
+{
+    use HasApiTokens;
+
+    ...
+}
+```
+
+You can optionally publish the protector config to set options regarding the storage, access and transmission of the files. 
+```bash
+artisan vendor:publish --tag=protector.config
+```
+
+Publish the protector database migration, and optionally modify it to work with your project.
+```bash
+php artisan vendor:publish --tag=protector.migrations
+```
+
+Run the migration on the client and server repository.
+```bash
+php artisan migrate
+```
+
+#### On the Client
+
+Run the following command to receive
+- the public key to give to your server admin
+- the private key to save in your .env file
+```bash
+php artisan protector:keys
+```
+
+Your server admin will then give you the token and server-url to save in your .env file.
+Unless specified otherwise in your software, the .env keys are
+```bash
+PROTECTOR_AUTH_TOKEN=
+PROTECTOR_SERVER_URL=
+```
+
+>Do not give your private key to anyone and keep it protected at all time
+
+See [Usage](#usage) on how to import the remote database.
```

Comment:
```suggestion
See [Usage](#Usage) on how to import the remote database.
```
Anchor link does not seem to work within Github. Seems like it is case sensitive ( see https://github.com/fefong/markdown_readme )

Reply:
https://github.com/cybex-gmbh/laravel-protector/tree/doc/provide-howto#readme it works, try it out here
github adds an anchor to headings and makes it kebap case

Reply:
taking my like and fleeing to hawaii